zio/zeekio/ztests/zeek-utf8-json.yaml

```diff
@@ -0,0 +1,80 @@
+zed: "*"
+
+# These test inputs are from src/util.cc in the Zeek source code as of Jan 2022.
+# See https://github.com/zeek/zeek/blob/3d9d6e953be5ae8b72336031b6e6659222f52cea/src/util.cc#L2521
```

Comment:
These are great tests, and I'm happy to have them here if that's your preference, but I do think they'd be more readable if moved to `escape_test.go`.

Reply:
As discussed, we can move them later or add another column to the Zeek TSV values as a comment explaining each test (taking the comments from the respective tests in the Zeek source code)